sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -207,6 +273,133 @@ class RocksDBSuite extends SparkFunSuite {
     }
   }
 
+  test("disallow concurrent updates to the same RocksDB instance") {
```

Comment:
This test seems not related to clean up change here? Looks like more related to RocksDB instance PR.

Reply:
Ah yea, this is the test for rollback.
Actually the original plan is expose `rollback` and `cleanup` in this PR. It should be a mistake for the last PR, I introduced the `rollback` without tests.

Reply:
okay